Github/package.json

```diff
@@ -10,9 +10,14 @@
   "dependencies": {
     "body-parser": "^1.18.3",
     "eslint": "^5.16.0",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-config-prettier": "^5.0.0",
+    "eslint-plugin-import": "^2.17.3",
+    "eslint-plugin-prettier": "^3.1.0",
     "express": "^4.16.3",
     "jquery": "^3.3.1",
     "path": "^0.12.7",
+    "prettier": "^1.18.2",
```

Comment:
Shouldn't these go to "devDependencies"? @poftadeh @manekenpix

Reply:
Yes, but linting is covered in #28, so I would recommend not committing these packages yet.

Reply:
I'll remove `prettier` then.